ibm/mas_devops/common_vars/casebundles/v8-230414-amd64.yml

```diff
@@ -25,6 +26,9 @@ mas_core_version:
 mas_assist_version:
   8.9.x: 8.6.3    # Updated
   8.10.x: 8.7.0   # No Update
+mas_hputilities_version:
+  8.9.x: 8.5.1    # Updated
+  8.10.x: 8.6.0   # Updated
```

Comment:
Is this PR also going to include hpu to be installed using airgap? If so can we update the PR description and PR title so it says that as well. It should also be a minor rather than patch and have changelog updates

Reply:
You would always need to mirror core (even if you did it long ago you can still run it again) so I don't think we need to add it into mirror-add-hputilities as well. If mirror-core was optional then I would say yes, but it isn't